File: problems/1137-height-checker/height-checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
const heightChecker = (heights) => {
    const originalHeights = [...heights];
    const sortedHeights = heights.sort((a, b) => a- b);
    let result = 0;
    for (let i = 0; i < heights.length; i++) {
        if (originalHeights[i] !== sortedHeights[i]) result ++;
    }
    return result;
};

File: problems/1137-height-checker/height-checker.py
import copy
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        new_list = list(heights)
        heights.sort()
        result = 0
        for i in range(len(heights)):
            if new_list[i] != heights[i]:
                result += 1
        
        return result